.gitignore

```diff
@@ -21,3 +21,4 @@ yarn-error.log*
 
 #TODO add .env to your gitignore
 
+/server/db/password.js
```

Comment:
I think it would be better to keep the password and similar secrets in a .env file that we ignore